resources/views/banking/transfers/mail.blade.php

```diff
@@ -1,9 +1,15 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <title>Bank Transfer</title>
-</head>
-<body>
-    Bank Transfer
-</body>
-</html>
\ No newline at end of file
+@component('mail::message')
+Good day,$bank_transfer
+
+This is your bank transfer details.
+
+@component('mail::table')
+|Transfer Date | Transfer Amount | Transfer Status | From Account | To Account |
+|:--------|:-------------|:-------|:---------| :-------|:-------| 
+| {{date_format($bank_transfer['created_at'],"Y/m/d");}} | {{$bank_transfer['amount']}} | {{$bank_transfer['status']}} | {{$bank_transfer['fromAccount']->chartOfAccount->account_name}} | {{$bank_transfer['toAccount']->chartOfAccount->account_name}} | 
```

Comment:
Make it something like this:
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |

```
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |
```